skills/csharp/tests/todoskill.tests/Flow/Utterances/AddToDoFlowTestUtterances.cs

```diff
@@ -74,6 +77,19 @@ public AddToDoFlowTestUtterances()
 
         public static string AddTaskWithContentAndCustomizeListType { get; } = "add history to my homework list";
 
+        public static string AddToDoActionName { get; } = "AddToDo";
```

Comment:
In my code, I unify these names together with the switch in RouteStep

Reply:
Nice design!